daemon/ZilliqaDaemon.cpp

```diff
@@ -230,8 +247,10 @@ vector<pid_t> ZilliqaDaemon::GetProcIdByName(const string& procName) {
   return result;
 }
 
-void ZilliqaDaemon::StartNewProcess() {
-  KillProcess();
+void ZilliqaDaemon::StartNewProcess(bool cleanPersistence) {
```

Comment:
For consistency with KillProcess signature change, should we pass `procName` here also, instead of using `programName`?